resolwe_bio/kb/migrations/0002_alter_fields.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9.11 on 2016-11-15 07:06
+from __future__ import unicode_literals
+
+import django.contrib.postgres.fields
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('resolwe_bio_kb', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='feature',
+            name='aliases',
+            field=django.contrib.postgres.fields.ArrayField(base_field=models.CharField(max_length=256), blank=True, default=[], size=None),
+        ),
+        migrations.AlterField(
+            model_name='feature',
+            name='name',
+            field=models.CharField(max_length=1024),
+        ),
+        migrations.AlterField(
+            model_name='feature',
+            name='sub_type',
```

Comment:
This field hasn't changed. Why is it in the migrations? @kostko is this the change that was missing from the migrations?